android/gradle/libs.versions.toml

```diff
@@ -66,7 +66,7 @@ okhttp = "4.12.0"
 okhttp-logging-interceptor = "4.12.0"
 orchestrator = "1.5.0"
 owasp = "8.2.1"
-p2p-lib = "0.6.9-SNAPSHOT"
+p2p-lib = "0.6.10-SNAPSHOT"
```

Comment:
@Rkareko Can we do this for the `opensrp-0.2.5` branch?

Reply:
The changes are already on `opensrp-0.2.5` branch